### ISSUE TITLE
Execution fails in Windows10

### ISSUE THREAD

**Issue by teddy-error** [butterproject/butter#26]
After building successfully with Grunt, we receive the following error when trying to launch
```
PS C:\Users\Dustin\butter> grunt start
Running "start" task

Running "exec:win" (exec) task
>> The system cannot find the path specified.
>> Exited with code: 1.
Warning: Task "exec:win" failed. Use --force to continue.

Aborted due to warnings.
PS C:\Users\Dustin\butter> grunt start --force
Running "start" task

Running "exec:win" (exec) task
>> The system cannot find the path specified.
>> Exited with code: 1.
Warning: Task "exec:win" failed. Used --force, continuing.
```

**Comment by teddy-error** [comment]
After looking into it further I found that the issue is with the file location in the Gruntfile on line 229:

```
        exec: {
            win: {
                cmd: '"build/cache/win/<%= nwjs.options.version %>/nw.exe" .'
```

Since cache is not a real directory inside of build we need to adjust this line to:

```
cmd: '"cache/<%= nwjs.options.version %>/win64/nw.exe" .'
```

**Comment by wvanbreukelen** [comment]
Got the exact same error.

**Comment by xaiki** [comment]
fixed in #34
